glusterd2/brick/types.go

```diff
@@ -18,6 +19,14 @@ const (
 	Arbiter
 )
 
+//const volflags
+const (
+	IsInUse          string = "reuse-bricks"
```

Comment:
If these are used in only one place, there's no point in these being constants. If you remove these, there will be less contusion between this variable here and `check.*` ones.

Reply:
it's used in multiple places, so kept it as the constant

Reply:
removed constant